### ISSUE TITLE
Drag-and-dropping multiple items into the scene at once in editor.

### ISSUE THREAD

**Issue by rbwldrf** [Facepunch/sbox-issues#5041]
### For?

S&Box

### What can't you do?

So I have a game that I've been working on that's supposed to take place in a 1:1 scale city and as a result, I have thousands of buildings in it, soon to be tens of thousands. Now, for various reasons, I have decided to split those buildings up into separate models (occlusion culling, LOD, physics don't work particularly well with objects of such scale and mesh density(?) with objects occasionally going through the buildings when it's all in a single collision mesh), so I've done that and imported them (with a really, _really_ hacky batch editor I had to write myself via OpenFBX because the batch editor is still in the backlog (thank God the VMDL format is human-readable)), but now when I select them all and try to put them in the scene, it only drops in a single one at a time, the one I click on when dragging the whole selection over.

### How would you like it to work?

I (and I can imagine a significant amount of users) want the ability to take the objects I've selected, drag them from the Asset Browser, and drop them all into the Scene simultaneously. I don't care if the engine or Qt will take minutes to process the request, so long as it eventually returns every object into the scene. How you make it work (do you put all objects into the centre of the mouse position, or do something to make it fancy) doesn't really matter to me, since all my meshes are offset in Blender and I'll just have to tare the Transform to its default values to get it set up.

### What have you tried?

I'm currently in the process of writing a script that spawns these objects in-game at launch, but I guess I kinda just expected it to work from the get-go. Like, I get that making a game engine with, like, a dozen people(?) is a _really_ challenging task, and it has an overwhelming lot of things that require attention, but this is just such an "essential" engine feature that I'm surprised this hasn't come up sooner.

**Comment by garrynewman** [comment]
TLDR, I want to drag multiple vmdl from the asset browser into the scene, right?

**Comment by garrynewman** [comment]
Where would they go? Would they all spawn on the same spot?

**Comment by rbwldrf** [comment]
As a rudimentary implementation, yes: for my scenario, it would be best for my thousands of models to spawn in the same spot, since they're already offset in the file.

You could make it fancy (spawn it in a grid, AI, etc.), but that sounds like a lot of work for a smidge of QOL.
